src/cmd/property.go

```diff
@@ -154,16 +201,16 @@ var updatePropertyDefinitionCmd = &cobra.Command{
 	Use:     "property-definition",
 	Aliases: []string{"propertydefinition", "propdef", "pd"},
 	Short:   "Update a property-definition",
-	Long:    `Update a property-definition`,
+	Long:    "Update a property-definition",
 	Example: fmt.Sprintf(`
-cat << EOF | opslevel update property-definition propdef3 -f -
+cat << EOF | opslevel create property-definition -f -
```

Comment:
looks like this was erroneously renamed from `update` to `create` - and i think this also means removeal of the argument is incorrect since the code below still uses `args[0]` and `cobra.ExactArgs(1)`